frontend/cypress/integration/authentication.spec.ts

```diff
@@ -0,0 +1,24 @@
+// TODO: Should correctly handle exceptions
+Cypress.on('uncaught:exception', (err, runnable) => {
+	// returning false here prevents Cypress from
+	// failing the test
+	return false
+})
+
+describe('Authentication', () => {
+	it('can login using email and password', () => {
+		cy.visit(`${Cypress.env('baseUrl')}/user/login`)
+
+		cy.get('#showPassword').click()
+		cy.get('#email').type('admin@windmill.dev')
+		cy.get('#password').type('changeme')
+		cy.get('.flex > .default-button').click()
+
+		cy.contains('Select a workspace')
```

Comment:
@rubenfiszel we should probably use i18n to avoid these kinds of string duplication.

Reply:
I thought i18n was mostly about internationalization. What do you have in mind ?